package.json

```diff
@@ -27,18 +27,19 @@
   "publishConfig": {
     "access": "public"
   },
+  "engines": {
+    "node": ">=18.15.0",
+    "npm": ">=9.5.0"
+  },
```

Comment:
**issue:** This is a shared dependency. We should not force the version here unless the features require it, but I don't think it is the case. Please let it open.

Reply:
agree, removed